microsoft/botframework-solutions
# Lamil/deployment updates

## Description

1. Added .bat files to parse .lu files into deploymentScripts folder for all languages and skills
2. Removed all .luis and .qna files and added to .gitignore
3. Added dispatch.lu for each language
3. Updated POI cancel route .lu to use patterns for redundant utterances to allow for ludown parsing of dispatch.lu (cannot have more that 15000 utterances for one intent)


# Notes
- This does not include post build steps due to increased build time
- Must run the .bat files before running msbotClone command

## Related Issue




fixes #21

## Testing Steps


1. Pull down code
2. Run update_deployment_scripts.bat 
3. Verify populated deployment script files for all languages in DeploymentScripts folder of VirtualAssistant
